pkmntypes.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, **kwargs):
 		self.EVs: list[int] = kwargs['EVs']
 		self.Nature: int = kwargs['Nature']
 		self.Stats: list[int] = kwargs['Stats']
-		self.StatusModifiers: list[int] = kwargs['StatusModifiers']
+		self.StatusModifiers: list[int] = kwargs['StatModifiers'] # TODO: clarify if this should read Stat or Status
```

Comment:
It should be `StatModifiers`
```suggestion
		self.StatModifiers: list[int] = kwargs['StatModifiers']
```